app/styles.module.css

```diff
@@ -13,4 +13,73 @@
   max-width: 350px;
   height: 100%;
   border: 1px solid black;
+  padding: 10px;
+}
+.headingInfo {
+  display: flex;
+  flex-direction: row;
+}
+.descriptionText {
+  font-size: 12px;
+  font-weight: 200;
+}
+div.headingInfo p {
+  font-size: 11px;
+  font-weight: 600;
+  padding-top: 10px;
+  padding: 5px;
+}
+
+div.headingInfo p:last-of-type {
+  margin-left: auto;
+  color: lightgray;
+  font-size: 12px;
+  font-weight: normal;
+}
```

Comment:
Can we use `justify-content: space-between` to separate the profile/username from the post date. And `align-items: center` to align items horizontally, that way we don't need to apply padding styles.